tests/Doctrine/Tests/DBAL/Driver/Mysqli/MysqliConnectionTest.php

```diff
@@ -28,11 +28,31 @@ protected function setUp()
             ->getMockForAbstractClass();
     }
 
+    protected function tearDown()
+    {
+        restore_error_handler();
```

Comment:
As you can see, at the end of the test there is a call to `set_error_handler($default_handler)`, which should essentially restore the error handler to the pre-test state. However, without `restore_error_handler` in tearDown, it causes 7 other tests to fail from some reason.

I have changed the tearDown to restore the error handler only for this test.